docs/cli/pull-from-upstream.md

```diff
@@ -12,13 +12,13 @@ upstream release. This command is meant to be called from **dist-git repository*
 
 
 ## Requirements
-
-* Upstream git repository on GitHub/GitLab.
-* Upstream release (git tag) where version in spec file is equivalent to
-  the name of the git tag (or `upstream_tag_template` is configured).
 * Packit config file placed in the dist-git repository.
 * Pagure API tokens for Fedora (or CentOS Stream) Dist-git.
 * Valid Fedora (or CentOS Stream) Kerberos ticket.
+* If `upstream_project_url` configured in Packit config:
+  * Upstream git repository on GitHub/GitLab.
+  * Upstream release (git tag) where version in spec file is equivalent to
+    the name of the git tag (or `upstream_tag_template` is configured).
```

Comment:
```suggestion
* If `upstream_project_url` is configured in Packit config (meaning the upstream project is expected to be a git repository):
  * Upstream git repository on GitHub/GitLab.
  * Upstream release (git tag) where version in spec file is equivalent to
    the name of the git tag (or `upstream_tag_template` is configured).
```

Since `upstream_project_url` does not mention git at all, I would try to be a bit more explicit about what it means to set/not set it.